src/gamekit.cs

```diff
@@ -376,10 +486,21 @@ interface GKLeaderboardSet : NSCoding, NSSecureCoding {
 		[Async]
 		void LoadLeaderboardSets ([NullAllowed] GKLeaderboardSetsHandler completionHandler);
 
+		[Internal]
+		[Deprecated (PlatformName.iOS, 14, 0, message: "Use 'LoadLeaderboardsWithCompletionHandler' instead.")]
+		[Deprecated (PlatformName.TvOS, 14, 0, message: "Use 'LoadLeaderboardsWithCompletionHandler' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10, 16, message: "Use 'LoadLeaderboardsWithCompletionHandler' instead.")]
+		[Deprecated (PlatformName.WatchOS, 7, 0, message: "Use 'LoadLeaderboardsWithCompletionHandler' instead.")]
 		[Export ("loadLeaderboardsWithCompletionHandler:")]
 		[Async]
 		void LoadLeaderboards ([NullAllowed] GKLeaderboardsHandler completionHandler);
 
+		[Internal]
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Export ("loadLeaderboardsWithHandler:")]
+		[Async]
+		void LoadLeaderboardsWithCompletionHandler ([NullAllowed] GKLeaderboardsHandler handler); // ideas for naming scheme here?
```

Comment:
Is there any explanation why there's a new method? I could be that we'd have to expose both, and let the customers start using the new one whenever they're ready (if the callback returns different data for instance).

Reply:
The API docs look exactly the same. The headers are slightly different:

```
/// Loads array with all classic leaderboards and current instances of recurring leaderboards for this leaderboardSet
/// Possible reasons for error:
/// 1. Communications problem
/// 2. Unauthenticated player
- (void)loadLeaderboardsWithHandler:(void(^)(NSArray<GKLeaderboard *> * __nullable leaderboards, NSError * __nullable error))handler API_AVAILABLE(ios(14.0), macos(10.16), tvos(14.0), watchos(7.0));

@end

@interface GKLeaderboardSet (Deprecated)

/// Loads array with all leaderboards for the leaderboardSet
/// Possible reasons for error:
/// 1. Communications problem
/// 2. Unauthenticated player
- (void)loadLeaderboardsWithCompletionHandler:(void(^__nullable)(NSArray<GKLeaderboard *> * __nullable leaderboards, NSError * __nullable error))completionHandler API_DEPRECATED("Use loadLeaderboardsWithHandler: instead.", ios(7.0, 14.0), tvos(9.0, 14.0), macosx(10.10, 10.16), watchos(3.0, 7.0));
```

Reply:
That didn't explain much :/

I think the safer option would be to not hide these API behind a version check, and instead expose both of them, because we don't know the difference between them. It could be that they're not interchangeable.

Reply:
The documentation does says to use the newer API instead of the deprecated one, but I see how it's better to give users more transparency if they're not the same 👍

https://developer.apple.com/documentation/gamekit/gkleaderboardset/1451810-loadleaderboardswithcompletionha